t/venom/myapp/test.yml

```diff
@@ -0,0 +1,11 @@
+name: Title of TestSuite
+testcases:
+
+  - name: Load database fixtures
+    steps:
+      - type: dbfixtures
+        database: mysql
+        dsn: root:inverse@(172.20.120.7:3306)/pf
+        folder: fixtures
```

Comment:
If you specify `fixtures` with `version: 2` on top of file, Venom will search a `fixtures` folder in `myapp/fixtures/`